contrib/gitian/README.md

```diff
@@ -96,7 +100,7 @@ sudo usermod -aG docker gitianuser
 Optionally add yourself to the docker group. Note that this will give docker root access to your system.
 
 ```bash
-sudo usermod -aG docker gitianuser
+sudo usermod -aG docker $USER
 ```
 
 Manual Building
```

Comment:
While you're at it, please delete line 109, which is redundant with line 114. This appears to be a copy/paste error that came in afbf05bc58a865818c4a437bf5a2ac76930f462f.
````
The instructions below use the automated script ...
````

Reply:
Removed.